package.json

```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/mozilla/webmaker-android/issues"
   },
   "dependencies": {
-    "webmaker-core": "https://github.com/mozilla/webmaker-core.git"
+    "webmaker-core": "git+https://github.com/mozilla/webmaker-core.git"
```

Comment:
this should not be necessary with an up to date `npm`, was this tested against npm 2.11.3?